hopsworks-api/src/main/java/io/hops/hopsworks/api/dela/DelaProjectService.java

```diff
@@ -250,7 +253,12 @@ public Response downloadDatasetKafka(@Context HttpServletRequest req, @Context S
     Dataset dataset = getDatasetByPublicId(publicDSId);
 
     String certPath = kafkaController.getKafkaCertPaths(project);
-    String brokerEndpoint = settings.getRandomKafkaBroker();
+    Optional<String> kafkaBrokerOpt = kafkaBrokers.getAnyKafkaBroker();
+    if (!kafkaBrokerOpt.isPresent()) {
```

Comment:
@kouzant we can use getOrElseThrow to simplify

Reply:
Done